GameOfLife/GameOfLife/Services/RulesApplier.cs

```diff
@@ -0,0 +1,138 @@
+﻿using GameOfLife.Interfaces;
+using GameOfLife.Models;
+using static GameOfLife.StringConstantsModel;
+
+namespace GameOfLife
+{
+    public class RulesApplier : IRulesApplier
+    {
+        private List<(int x, int y)> _cellsToDie = new List<(int x, int y)>();
+        private List<(int x, int y)> _cellsToBeBorn = new List<(int x, int y)>();
+        private int neighboursCount;
+
+        /// <summary>
+        /// Method to determine which cells die and which are reborn, judging by the number of alive neighbours.
+        /// </summary>
+        /// <param name="gameField">An instance of the GameFieldModel class that stores the game field and its properties.</param>
+        /// <param name="disableWrappingAroundField">Parameter that shows if field's wrapping around is enabled.</param>
+        public void DetermineCellsDestiny(GameFieldModel gameField, bool disableWrappingAroundField)
+        {
```

Comment:
Logic for this method is not easy to read. Split it into three methods. This stays as master method which depending on disableWrappingAroundField calls either method for wrapped world or other for non wrapped world.